### DISCUSSION TITLE
Enable packer_palette:NCAA_expanded in XML script?

### DISCUSSION THREAD

**Discussion by tsamdin** [RosettaCommons/rosetta#110] [RosettaScripts] [answered]
Is it possible to enable the NCAA_expanded library in the XML script, or is it a requirement that a flag always be passed for -packing:packer_palette:NCAA_expanded at the command line?

**Comment by vmullig** [comment, marked as answer]
No, you can also do it in RosettaScripts in a `<PACKER_PALETTES> ... </PACKER_PALETTES>` section of the script.  See this page:

https://docs.rosettacommons.org/docs/latest/rosetta_basics/structural_concepts/PackerPalette#usage_rosettascripts_example-rosettascripts-xml

**Comment by roccomoretti** [comment]
Marking this as "closed", as I hope your question has been answered. If that assessment is incorrect, feel free to re-open or open up a new discussion with more details.
